src/main/java/org/ml_methods_group/ui/EmptyResultNotification.java

```diff
@@ -0,0 +1,21 @@
+package org.ml_methods_group.ui;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import org.jetbrains.annotations.NotNull;
+
+public class EmptyResultNotification extends Notification {
+    private final static String GROUP_DISPLAY_ID = "Search For Refactorings";
```

Comment:
this should not be hardcoded, but put into ArchitectureReloadedBundle.properties and accessed via `ArchitectureReloadedBundle.message()`